app/controllers/api/v2/reports_controller.rb

```diff
@@ -39,7 +39,7 @@ def show
       def create
         import_params = { :report => params[:report] }
         task = ForemanTasks.async_task Actions::Foreman::Report::Import, import_params, resource_class, detected_proxy.try(:id)
-        process_success _('Report import has been enqueued, please see the task %s for more details') % task.id
+        process_success _('Report import has been enqueued, please see the task %s for more details') % task.id, :accepted
```

Comment:
bonus points by @ekohl (he promised it on the ticket 😄) if you provide the URL along with the task ID